MovieApp/MovieApp/FeedTableViewCell.h

```diff
@@ -0,0 +1,19 @@
+//
+//  FeedTableViewCell.h
+//  MovieApp
+//
+//  Created by user on 23/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CustomTableViewCell.h"
+
+@interface FeedTableViewCell : CustomTableViewCell
+
+@property (weak, nonatomic) IBOutlet UILabel *labelHeadline;
```

Comment:
I prefer calling properties `headlineLabel`, with type indicator as suffix, instead of prefix.